src/lib/pages/home/index.tsx

```diff
@@ -38,18 +38,16 @@ const Home = () => {
   const [repositories, setRepositories] = useState([]);
   const fetchRepositories = async (value: string) => {
     const response = await fetch(
-      `https://api.github.com/search/repositories?q=${value}%20in:name%20titanium%20in:topics%20language:objc+language:swift+language:java+language:kotlin+language:javascript&sort=updated&order=desc`,
+      `/api/modules?search=${encodeURIComponent(value)}`,
       {
         headers: {
           "User-Agent": "Titanium Module Search",
```

Comment:
At this point, might as well axe all of the `headers` since the `User-Agent` is pointless.

Reply:
Fixed